data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/ConnectionConfiguration.java

```diff
@@ -47,11 +50,11 @@ public boolean isInsecure() {
     return insecure;
   }
 
-  @AssertTrue(message = "Certificate file path and certificate content both are configured. " +
+  @AssertTrue(message = "cert and certificate both are configured. " +
```

Comment:
What could we do to push the validation up to here? Say, the file is missing or the it is not a valid certificate? Maybe this is for a follow-on PR, but it would be a nice improvement for users.

Reply:
Did some refactoring and it should be floated up.